.coafile

```diff
@@ -1,7 +1,7 @@
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = **/*.rst, README.md
+files = **/*.rst
```

Comment:
this shouldn't be, but if this is the case right now go with it

looks like we have a serious definition problem with globs right now, will have to think about that and discuss that somewhere else later.